### ISSUE TITLE
0,0 degrees Celsius after waking up laptop

### ISSUE THREAD

**Issue by Vistaus** [ilgarmehmetali/budgie-weather-applet#4]
Every time I wake up my laptop (that is, opening the lid to wake it from suspend mode), the weather applet shows 0,0 degrees and I have to go to the applet settings and click "Update now" before it shows the temperature again.

**Comment by Vistaus** [comment]
Thanks for fixing it! :)

**Comment by ilgarmehmetali** [comment]
i think 0,0 might be caused when weather data couldnt gets downloaded too

**Comment by ilgarmehmetali** [comment]
Still persists. Will look into this again.

**Comment by Vistaus** [comment]
You're right. At first it appeared to be fixed but it seems to have resurfaced. Thanks for looking into it again :)

**Comment by ilgarmehmetali** [comment]
Should be fixed with https://github.com/ilgarmehmetali/budgie-weather-applet/commit/517faa6cc1d779dffaa22d1bb17926c8ad2ae787


probably... :)

**Comment by Vistaus** [comment]
Yup, looks like it's fixed again. Thanks! :smile:
